layouts/partials/feedback.html

```diff
@@ -15,9 +15,9 @@
 </style>
 <div class="d-print-none">
 <h2 class="feedback--title">Feedback</h2>
-<p class="feedback--question">Was this page helpful?</p>
-<button class="btn btn-primary mb-4 feedback--answer feedback--answer-yes">Yes</button>
-<button class="btn btn-primary mb-4 feedback--answer feedback--answer-no">No</button>
+<p class="feedback--question">{{- T "feedback_question" -}}</p>
+<button class="btn btn-primary mb-4 feedback--answer feedback--answer-yes">{{- T "feedback_positive" -}}</button>
+<button class="btn btn-primary mb-4 feedback--answer feedback--answer-no">{{- T "feedback_negative" -}}</button>
```

Comment:
Hmm, maybe we should start using `i18n` instead of its alias `T`? Ok to use the alias for now, since it what is used elsewhere in Docsy.

Ref: https://gohugo.io/functions/i18n/

Reply:
> Hmm, maybe we should start using `i18n` instead of its alias `T`?

I like the idea and changed all occurrences of `T` to `i18n`.